### DISCUSSION TITLE
File Manager overwrite file

### DISCUSSION THREAD

**Discussion by jackthebean** [FossifyOrg/General-Discussion#482] [General]
Just saved a new version of a file to a folder using File Manager.  I was expecting it to tell me that there was already a file with that name and to ask me to confirm overwriting it.  It did not nor did it give it a new name eg name - copy or name (1) but instead just overwrote the original.  In this case that was fine but generally surely it should provide a warning about an overwrite etc.
Thanks,

**Comment by Aga-C** [comment]
Please create a bug report for it.
